desktop/src/com/frostwire/gui/bittorrent/CreateTorrentDialog.java

```diff
@@ -99,6 +94,40 @@
     private String _invalidTrackerURL;
     private JButton _buttonClose;
 
+    private JComboBox pieceSizeComboBox;
+    private int pieceSize;
+    private static final int _1024BYTES = 1024;
```

Comment:
I think it's self explanatory because the enum clearly says you're working with`KB`.
If you still want to keep it, you could name it `KB` instead.